javascripts/discourse/widgets/dc-topic-title.js.es6

```diff
@@ -5,7 +5,43 @@ import { h } from "virtual-dom";
 export default createWidget("dc-topic-title", {
   tagName: "div.dc-topic-title",
 
+  buildCategory(attrs) {
```

Comment:
What are the issues you encountered for not to use a built-in widget to render the category?
_I just wanted to make sure you evaluate the possibility of adding a tag using Discourse core but you couldn't_

Reply:
Yes, actually that's the case. This was the only way I found to perform this task, we may do some pair programming to considerate some other approach. What do you think?

Reply:
We can check it out afterwards as a side activity to understand a bit more the `whys` of this. _(I think I have an idea)_ but in any case for now I think we are good to go!